docs/tutorials/ad-hoc-simulation/rosbag-replay-simulation.md

```diff
@@ -4,20 +4,22 @@
 
 1. Download and unpack a sample map.
 
-   - Click [here](https://drive.google.com/file/d/1A-8BvYRX3DhSzkAnOcGWFw5T30xTlwZI/view?usp=sharing) to download.
-   - Unpack it by running the following command.
+   - You can also download [the map](https://drive.google.com/file/d/1A-8BvYRX3DhSzkAnOcGWFw5T30xTlwZI/view?usp=sharing) manually.
 
    ```bash
-   unzip -d ~/Downloads/ ~/Downloads/sample-map-rosbag.zip
+   wget -O ~/autoware/sample-map-rosbag.zip 'https://docs.google.com/uc?export=download&id=1A-8BvYRX3DhSzkAnOcGWFw5T30xTlwZI'
+   unzip -d ~/autoware/ ~/autoware/sample-map-rosbag.zip
    ```
 
 2. Download the sample rosbag files.
 
-   - Click [here](https://drive.google.com/file/d/1VnwJx9tI3kI_cTLzP61ktuAJ1ChgygpG/view?usp=sharing) to download.
-   - Unpack it by running the following command.
+   - You can also download [the rosbag files](https://drive.google.com/file/d/1VnwJx9tI3kI_cTLzP61ktuAJ1ChgygpG/view?usp=sharing) manually.
 
    ```bash
-   unzip -d ~/Downloads/ ~/Downloads/sample-rosbag.zip
+   # The file exceeds 100 MB and we need to confirm before downloading
+   wget --save-cookies /tmp/cookies.txt 'https://docs.google.com/uc?export=download&id=1VnwJx9tI3kI_cTLzP61ktuAJ1ChgygpG' -O- | sed -rn 's/.*confirm=([0-9A-Za-z_]+).*/\1/p' > /tmp/confirm.txt
+   wget --load-cookies /tmp/cookies.txt -O ~/autoware/sample-rosbag.zip 'https://docs.google.com/uc?export=download&id=1VnwJx9tI3kI_cTLzP61ktuAJ1ChgygpG&confirm='$(</tmp/confirm.txt)
```

Comment:
In this complicated case, I believe we should use a third-party tool such as https://github.com/wkentaro/gdown.

Reply:
That's a cool tool! Thanks for sharing.
I guess we need to install it before using it. Where do you think is a good place to put the download command?
Maybe I can mention it in every tutorial before trying to download files from Google Drive.

Reply:
Considering we support both Ansible and manual installation, I think adding a role here is good.
https://github.com/autowarefoundation/autoware/tree/main/ansible/roles

Reply:
I found that autoware has already installed `gdown` [here](https://github.com/autowarefoundation/autoware/blob/210f00137ae3adc53c63b63f1a9a486cb8ac569d/ansible/roles/autoware_core/tasks/main.yaml). 😆

Reply:
Oh, I forgot about that. :rofl: